dbench.yaml

```diff
@@ -24,22 +20,27 @@ spec:
     spec:
       containers:
       - name: dbench
-        image: logdna/dbench:latest
+        image: longhorn/dbench:latest
+        #image: boknowswiki/dbench:longhorn
         imagePullPolicy: Always
+        securityContext:
+          # Need this for fio be able to invalid the cache
+          privileged: true
+        #command: ["/bin/sh", "-c", "while true; do :; done"]
```

Comment:
And this line.

Reply:
Do you mean to enable the command? I think it will over write the docker image command which runs the `docker-entrypoint.sh`.

Reply:
No, I meant we should remove this line. I don't understand why it's here.

Reply:
I have submitted another [PR](https://github.com/longhorn/dbench/pull/2) to remove them and correct the Longhorn docker hub hander.